survey_multi_company/security/survey_security.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="survey_survey_company_rule" model="ir.rule">
+        <field name="name">Survey Multi Company</field>
+        <field name="model_id" ref="survey.model_survey_survey" />
+        <field
+            name="domain_force"
+        >['|',('company_id','=',False),('company_id', 'in', company_ids)]</field>
```

Comment:
**suggestion:** Simplify it and improve performance:

```suggestion
        >[('company_id','in',[False] + company_ids)]</field>
```

Reply:
cool :) thanks